src/pages/about/About.scss

```diff
@@ -0,0 +1,8 @@
+ul {
+    list-style: none;
+    padding: 0;
+}
+
+li img {
+    width: 5%;
```

Comment:
did you check that on different screen sizes?

Reply:
is there something wrong you can see now?

Reply:
changed img width to `1.9em`
![image](https://github.com/hasadna/open-bus-map-search/assets/1336862/1edb5494-675a-45c3-a4eb-573d00068564)

Reply:
I didn't check how it looks, only the code and screenshot.
consider adding `margin-right` and bullets to list